doc/classes/FontConfig.xml

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FontConfig" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		FontConfig class font configuration used to draw text.
+	</brief_description>
+	<description>
+		[codeblocks]
```

Comment:
Could use more description before the example code to better explain what FontConfig is used for exactly, and how it differs from and interacts with Font.

Reply:
Added a bit more usage info to the [Font] and [FontConfig].